src/librustc/ich/impls_hir.rs

```diff
@@ -654,11 +654,12 @@ impl<'a> HashStable<StableHashingContext<'a>> for ast::Ident {
                                           hcx: &mut StableHashingContext<'a>,
                                           hasher: &mut StableHasher<W>) {
         let ast::Ident {
-            ref name,
-            span: _ // Ignore this
+            name,
+            span,
         } = *self;
 
         name.hash_stable(hcx, hasher);
+        span.hash_stable(hcx, hasher);
```

Comment:
cc @michaelwoerister (making sure you see this)

Reply:
mw previously confirmed the span should be hashed here - https://github.com/rust-lang/rust/pull/49154#discussion_r175555435